odxtools/diaglayer.py

```diff
@@ -238,11 +152,23 @@ def data_object_properties(self) -> NamedItemList[DopBase]:
         """
         return self._data_object_properties
 
+    #######
+    # </stuff subject to value inheritance>
+    #######
+
+    #######
+    # <communication parameters>
+    #######
+
     @property
     def communication_parameters(self) -> NamedItemList[CommunicationParameterRef]:
         """All communication parameters including inherited ones."""
```

Comment:
If this class is a wrapper for "Diagnostic Layer" including "inherited Diagnostic Layers", then this class is no longer a "Layer", it would be more appropriate to call it a "Diagnostic Stack" (since it includes multiple layers)

Reply:
while I don't disagree, the ODX spec calls the logical view on such an object -- i.e., including inheritance -- a "layer" (cf. section 7.3.2). I think we should only deviate from the spec if there is a *very* good reason for it...